Packs/Akamai_WAF/Integrations/Akamai_WAF/README.md

```diff
@@ -1683,4 +1683,43 @@ Gets the status of a pending change.
                 }
               }
 }}}}
-```
\ No newline at end of file
+```
+### akamai-get-cps-enrollment-by-id
+
+***
+Get an enrollment in CPS by enrollment id
+
+#### Base Command
+
+`akamai-get-cps-enrollment-by-id`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| enrollment_id | Enrollment on which to perform the desired operation. | Required | 
+
+#### Context Output
+
+There is no context output for this command.
+### akamai-cancel-cps-change
+
+***
+Cancels a pending change on CPS.
+
+#### Base Command
+
+`akamai-cancel-cps-change`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| change_id | The change for this enrollment on which to perform the desired operation. Default is 0. "change_path" is used. Default is 0. | Required | 
+| enrollment_id | Enrollment on which to perform the desired operation. Default is 0. "change_path" is used. Default is 0. | Required | 
+| change_path | Change path on which to perform the desired operation. Sample: /cps/v2/enrollments/100000/changes/88888888. Note: change_path is not listed in the reference as a parameter. However it can be extracted directly from "list_enrollments_command". This should be the most common useage when generate RestAPI's URL. | Optional | 
```

Comment:
Not sure if this is needed:
Note: change_path is not listed in the reference as a parameter.

Instead, i would remove the first part of the note, and only say:
The change path can be extracted directly from "list_enrollments_command". This should be the most common usage when generating the RestAPI's URL."